src/v2/view-builder/components/AuthenticatorEnrollOptions.js

```diff
@@ -16,19 +16,26 @@ import hbs from 'handlebars-inline-precompile';
 
 const AuthenticatorRow = View.extend({
   className: 'authenticator-row clearfix',
-  template: hbs('\
-        <div class="authenticator-icon-container">\
-          <div class="factor-icon authenticator-icon {{iconClassName}}">\
-          </div>\
-        </div>\
-        <div class="authenticator-description">\
-          <h3 class="authenticator-label no-translate">{{label}}</h3>\
-          {{#if description}}\
-            <p class="authenticator-description--text">{{description}}</p>\
-          {{/if}}\
-          <div class="authenticator-button" {{#if buttonDataSeAttr}}data-se="{{buttonDataSeAttr}}"{{/if}}></div>\
-        </div>\
-      '),
+  template: hbs`
+        <div class="authenticator-icon-container">
+        {{#if logoUri}}
+          <div class="custom-auth-icon">
```

Comment:
can `factor-icon` and `authenticator-icon` css class be used here as well? it may already have the needed css values.

Reply:
i had to use `<img>` earlier so i cannot really reuse existing classes. Now that requirement has changed, i have used `background-image` with existing classes